src/modules/Task.js

```diff
@@ -0,0 +1,29 @@
+export default class Task {
+  constructor(name, dueDate = 'No date') {
+    this.name = name;
+    this.dueDate = dueDate;
+  }
+
+  setName(name) {
+    this.name = name;
+  }
+
+  getName() {
+    return this.name;
+  }
+
+  setDate(dueDate) {
+    this.dueDate = dueDate;
+  }
+
+  getDate() {
+    return this.dueDate;
+  }
+
+  getDateFormatted() {
+    const day = this.dueDate.split('/')[0];
+    const month = this.dueDate.split('/')[1];
+    const year = this.dueDate.split('/')[2];
+    return `${month}/${day}/${year}`;
+  }
+}

```

Comment:
- [ ] Each task should at least contain a title, description, due date, and priority. Your task is missing the description and priority fields.